test/unit/Model.js

```diff
@@ -1167,7 +1170,11 @@ describe("Model", () => {
 							}
 						},
 						"TableName": "User",
-						"ProjectionExpression": "id, name"
+						"ProjectionExpression": "#id, #name",
+						"ExpressionAttributeNames": {
+							"#id": "id",
+							"#name": "name"
+						}
```

Comment:
Will need to update the tests with my requested changes.

Reply:
Fixed